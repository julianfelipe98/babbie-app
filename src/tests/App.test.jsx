const { shallow } = require("enzyme");
const { App } = require("../App");

describe("App component tests", () => {
  test("should render <App/> in the correct way", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
