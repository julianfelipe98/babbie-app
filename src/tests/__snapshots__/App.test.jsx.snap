// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component tests should render <App/> in the correct way 1`] = `
<div>
  <NavBar />
  <Hero />
  <Cards />
  <Banner
    btn="Descargar ahora"
    id="2"
    paragraph="Disponible para tu dispositivo Android. comienza la experiencia ahora."
    styleBtn="download-btn"
    subtitle1="Descarga la aplicacion ahora!"
  />
  <Footer />
</div>
`;
